### PR TITLE
Team Services Added

### PR DESCRIPTION
**Pull request by luericlueric**
Adding team services. 

team resolver for logosrc and photosrc to display "No logo/photo uploaded" when it's null / empty.

### CODE REVIEW

File: src/services/team/team.schema.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  
+
+export type Team = Static<typeof teamSchema>
+export const teamValidator = getValidator(teamSchema, dataValidator)
+export const teamResolver = resolve<Team, HookContext>({
+  photosrc: async (value) => {
+    if (value != null && value.trim() !== "") {
+      return `http://svcsa.org/uploads/${value}`;
+    }
+    return "No Image Uploaded!";
```
Comment (umliuxin) [inline src/services/team/team.schema.ts]: should be return null? 
and make the photosrc to be optional?

File: src/services/team/team.schema.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  },
+
+  logosrc: async (value) => {
+    if (value != null && value.trim() !== "") {
+      return `http://svcsa.org/uploads/${value}`;
+    }
+    return "No logo Uploaded!"
```
Comment (umliuxin) [inline src/services/team/team.schema.ts]: same here

**Review by umliuxin — COMMENTED** [review]
looks pretty good!

File: src/services/team/team.schema.ts (hunk 1 above)
Comment (luericlueric) [reply src/services/team/team.schema.ts]: photosrc set to optional

**Comment by umliuxin** [comment]
```
services
-track-field
---tf-player
---tf-team
---...
-basketball
---bb-player
---bb-team
---...
```

**Comment by umliuxin** [comment]
https://github.com/umliuxin/svcsa-tf-api#services-structure

**Review by umliuxin — APPROVED** [review]

**Pull request merged**